<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLogin</class>
 <widget class="QDialog" name="CLogin">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_login">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_set">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_status">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_username">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>211</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_userpassword">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>211</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>79</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userpassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_curtime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_logo">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_about">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_topbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_warnning">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lb_topbar</zorder>
  <zorder>pb_login</zorder>
  <zorder>pb_set</zorder>
  <zorder>cb_status</zorder>
  <zorder>le_username</zorder>
  <zorder>le_userpassword</zorder>
  <zorder>username</zorder>
  <zorder>userpassword</zorder>
  <zorder>lb_curtime</zorder>
  <zorder>lb_logo</zorder>
  <zorder>pb_about</zorder>
  <zorder>pb_close</zorder>
  <zorder>lb_warnning</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
